content/community/events.page.tsx

```diff
@@ -56,10 +54,9 @@ const events = [
     link: "https://www.meetup.com/xrpl-community/events/281980446/",
     location: "Atlanta, GA",
     date: "Saturday, November 27, 2021",
-    image: "event-meetup-alanta@2x.jpg",
+    image: require("../static/img/events/event-meetup-alanta@2x.jpg"),
     end_date: "November 27, 2021",
   },
-
   {
     name: "XRPL Community Meetup: San Francisco",
```

Comment:
I'm not sure exactly where this comes from, but apparently the name being a duplicate is causing a React error.

```
Warning: Encountered two children with the same key, `XRPL Community Meetup: San Diego`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.
```

There's a similar error for `XRPL Accelerator Demo Day` as well.